### PR TITLE
[MIG] Migration of account_invoice_merge_purchase to 10.0

### PR DESCRIPTION
**Pull request by adrienpeiffer**
- [x] depends on #238

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This has been recently migrated to 9.0, so little changes are needed from that version. Please respect commit history following migration guide: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0

**Comment by adrienpeiffer** [comment]
@pedrobaeza It's done.

File: account_invoice_merge_purchase/README.rst
Comment (elicoidal) [inline account_invoice_merge_purchase/README.rst:10]: Please add a period.

File: account_invoice_merge_purchase/README.rst
Comment (elicoidal) [inline account_invoice_merge_purchase/README.rst:16]: Please use enumerated list ("#. ")

File: setup/account_invoice_merge_purchase/odoo/__init__.py
Comment (elicoidal) [inline setup/account_invoice_merge_purchase/odoo/__init__.py:1]: do you really need to have those files?

**Review by elicoidal — APPROVED** [review]
nitpicking.
Do you need the setup directory?

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please check Travis

**Comment by adrienpeiffer** [comment]
@pedrobaeza Thanks. account_invoice merge is not yet merged.

**Comment by pedrobaeza** [comment]
Yeah, that's right! OK, let's put this as WIP. You have already my code review approval anyway :wink:

File: setup/account_invoice_merge_purchase/odoo/__init__.py
Comment (adrienpeiffer) [reply setup/account_invoice_merge_purchase/odoo/__init__.py:1]: yes :smile:

**Comment by MiquelRForgeFlow** [comment]
@adrienpeiffer Can you update this PR, please?

**Comment by JordiBForgeFlow** [comment]
@adrienpeiffer Now that  account_invoice merge there are now some errors in Travis.

**Comment by pedrobaeza** [comment]
A rebase is needed in this PR

**Comment by adrienpeiffer** [comment]
Rebased

**Comment by pedrobaeza** [comment]
Travis is still failing. Do you know why?

**Review by benwillig — APPROVED** [review]
LGTM :+1:

**Comment by LoisRForgeFlow** [comment]
@adrienpeiffer Any update here? Tests are failing.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Tests of this module are failing

**Comment by rousseldenis** [comment]
@aheficent I'm on it

**Comment by rousseldenis** [comment]
@aheficent Done. Waiting for travis :sweat_smile:

**Pull request merged**
